lumapps/api/lumapps_jwt.py

```diff
@@ -23,8 +23,8 @@
     "dot-lumapps-social.appspot.com": "https://login.dev.lumapps.com/v1/jwks",
     "dot-lumapps-dev.appspot.com": "https://login.dev.lumapps.com/v1/jwks",
     "dot-lumapps-sandbox.appspot.com": "https://login-sandbox.dev.lumapps.com/v1/jwks",
-    "dot-lumapps-beta.appspot.com": "https://login.beta.lumapps.com/v1/jwks",
```

Comment:
I think you can remove these too, we have now lumapps-csbx-ms, lumapps-csbx-eu, lumapps-csbx-us

Reply:
updated!